server/views/match/placementRequests/occupancyView/view.njk

```diff
@@ -39,14 +42,66 @@
     }) }}
 
     <section>
-        <h2 class="govuk-heading-l">View availability and book your placement
-            for {{ formatDuration(durationDays) }}
+        <h2 class="govuk-heading-l">View availability for {{ formatDuration(durationDays) }}
             from {{ formatDate(startDate, { format: 'short' }) }}</h2>
 
-        {{ govukNotificationBanner({
-            html: occupancySummaryHtml,
-            classes: 'govuk-notification-banner--full-width-content'
-        }) }}
+        <div class="search-and-filter">
+            <form action="{{ paths.v2Match.placementRequests.spaceBookings.viewSpaces({ id: placementRequest.id }) }}"
+                  method="get">
+                <h3 class="govuk-heading-m">Filters</h3>
+
+                <input type="hidden" name="premisesId" value="{{ premisesId }}">
+                <input type="hidden" name="premisesName" value="{{ premisesName }}">
```

Comment:
So, these are needed because the POST handler currently uses them. We should be able to rewrite this so the form posts to the url _including the query string_, which would make all these available to the handler, but I'm going to keep that as a separate refactor. That handler definitely needs looking at, as at the moment it returns errors one by one, for instance, which would be jarring to the user.